.github/workflows/release-upload.yaml

```diff
@@ -0,0 +1,39 @@
+name: Post Release Upload
```

Comment:
We need a few lines of documentation at the top here explaining what the reason for this workflow is.
What do we want, why do we want it, how do we achieve it? All three questions should be answered.